src/migration/1715679235259-AdvancedSettingsUpgrade.ts

```diff
@@ -0,0 +1,53 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AdvancedSettingsUpgrade1715679235259
+  implements MigrationInterface
+{
+  name = 'AdvancedSettingsUpgrade1715679235259';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    // DELETE OLD GROUP ADVANCED TYPES
+    await queryRunner.query(
+      `DELETE FROM advanced_settings WHERE "group" = 'advanced'`,
+    );
```

Comment:
This will cascade and delete `feed_advanced_settings` for current user feeds? Is that what we want?

Reply:
Yep :) 
They not really used anyway only in the old resolver, just cleans up neatly